File: fuzzer/compiled_programs/gen_JVIdmZrJUSY.r
library(tidyr)
suppressMessages(library(dplyr))
library(e1071) # SVM implementation
library(class) # KNN implementation
library(rpart) # DecisionTree implementation
library(nnet) # MLP implementation
suppressMessages(library(caret))

OIgtDTG <- read.csv("../datasets/Iris__Species.csv")
OIgtDTG <- OIgtDTG %>% drop_na()

OIgtDTG_X <- subset(OIgtDTG, select = -Species)
OIgtDTG_Y <- as.factor(OIgtDTG$Species)

OIgtDTG_X <- apply(OIgtDTG_X, 2, function(x) x / max(abs(x)))

ZCYMKxw <- read.csv("../datasets/titanic__Survived.csv")
ZCYMKxw <- ZCYMKxw %>% drop_na()

ZCYMKxw_X <- subset(ZCYMKxw, select = -ncol(ZCYMKxw))
ZCYMKxw_Y <- as.factor(ZCYMKxw[,ncol(ZCYMKxw)])

File: fuzzer/compiled_programs/gen_JVIdmZrJUSY.py
from sklearn import *
import pandas as pd

OIgtDTG = pd.read_csv("../datasets/Iris__Species.csv")
OIgtDTG = OIgtDTG.dropna()

OIgtDTG_Y = OIgtDTG["Species"]
OIgtDTG = OIgtDTG.drop(columns=["Species"])

OIgtDTG_scaler = preprocessing.AbsMaxScaler()
OIgtDTG = OIgtDTG_scaler.fit_transform(OIgtDTG)

ZCYMKxw = pd.read_csv("../datasets/titanic__Survived.csv")
ZCYMKxw = ZCYMKxw.dropna()

ZCYMKxw_Y = ZCYMKxw.iloc[:,-1]
ZCYMKxw = ZCYMKxw.iloc[:, :-1]

